src/main/java/com/northconcepts/datapipeline/foundations/examples/time/DetectDate_TimePatternsInStrings.java

```diff
@@ -0,0 +1,48 @@
+package com.northconcepts.datapipeline.foundations.examples.time;
+
+import com.northconcepts.datapipeline.core.DataException;
+import com.northconcepts.datapipeline.foundations.time.DateTimePatternMatch;
+import com.northconcepts.datapipeline.foundations.time.DateTimePatternDetector;
+
+import java.time.temporal.TemporalQueries;
+import java.util.List;
+
+public class DetectDate_TimePatternsInStrings {
```

Comment:
Let's drop the underscore from the class name.

Reply:
Done